Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -126,6 +126,19 @@ public struct WebView: UIViewRepresentable {
             UIApplication.topViewController()?.present(alertController, animated: true, completion: nil)
         }
 
+        public func webView(
+            _ webView: WKWebView, createWebViewWith configuration: WKWebViewConfiguration,
+            for navigationAction: WKNavigationAction,
+            windowFeatures: WKWindowFeatures
+        ) -> WKWebView? {
+            if navigationAction.targetFrame == nil, let url = navigationAction.request.url {
+                if UIApplication.shared.canOpenURL(url) {
+                    UIApplication.shared.open(url)
+                }
+            }
+            return nil
+        }
```

Comment:
It's working fine and addressing the underlying issue. But as per the documentation this delegate method is used to `Creates a new web view.` but I guess it's for the app and not for the external like opening a link in safari.

In the production app, it's handled in the delegate method. `decidePolicyFor navigationAction`

```
public func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
            let isWebViewDelegateHandled = !(webViewDelegate?.webView(webView, shouldLoad: navigationAction.request) ?? true)
            if isWebViewDelegateHandled {
                decisionHandler(.cancel)
            } else {
                switch navigationAction.navigationType {
                case .linkActivated:
                    if let url = navigationAction.request.url, UIApplication.shared.canOpenURL(url) {
                        UIApplication.shared.open(url, options: [:], completionHandler: nil)
                    }
                    decisionHandler(.cancel)
                default:
                    decisionHandler(.allow)
                }
            }
        }

Reply:
`target=blank` is forcing to open link in in new window - it's why `createWebViewWith` method is working here. But you're right and we could use `decidePolicyFor navigationAction` too. I'll a little refactor this. Thank you